Mage.Sets/src/mage/cards/b/BeneathTheSands.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Open a dos prompt and run it that way.  It will ask for the name of the card (but as it's already implemented, you may want to try a new one)